src/main/java/Main.java

```diff
@@ -1,19 +1,28 @@
 import model.BlindHorizontal;
-import service.HorizontalService;
-import service.HorizontalServiceImpl;
-import service.PriceCatalog;
-import service.PriceCatalogImpl;
+import model.BlindVertical;
+import service.blindService.*;
+import service.catalog.*;
+import service.fileService.*;
```

Comment:
It is about class Main naming. Let's rename it to name of program, Calculator e.g.